WasatchNET/EEPROM.cs

```diff
@@ -1154,7 +1154,7 @@ public bool read()
 
                 //!!!!!!!
                 //!!!!!!!
-                laserExcitationWavelengthNMFloat = 0.0f;//830.0f;//ParseData.toFloat(pages[3], 36);
+                laserExcitationWavelengthNMFloat = 785.0f;//830.0f;//ParseData.toFloat(pages[3], 36);

```

Comment:
I don't like this, but accepting as it will be replaced as soon as EEPROM read/write is supported.